fubectl.source

```diff
@@ -34,10 +36,12 @@ alias kp="xdg-open 'http://localhost:8001/api/v1/namespaces/kube-system/services
 
 # [kwatch] watch resource
 kwatch() {
-    if _isClusterSpaceObject $1 ; then
-        kubectl get "${1}" | _inline_fzf | awk '{print $1}' | xargs watch kubectl get "${1}"
-    else
-        kubectl get "${1}" --all-namespaces | _inline_fzf | awk '{print $1, $2}' | xargs watch kubectl get "${1}" -n
+    if _isObjectSpecified $1 ; then
```

Comment:
Can we avoid the big nesting braces?
I'd prefer something like `[ -z "$1" ] && fail "xyz not specified."` this allows a more specific error message and uses a lot less code.